BSP/Inc/BSP_Contactors.h

```diff
@@ -0,0 +1,12 @@
+#ifndef __BSP_CONTACTORS_H
+#define __BSP_CONTACTORS_H
+
+#include "common.h"
+#include "config.h"
+#include <bsp.h>
+
+void BSP_Contactors_Init(void);
+void read_Contactor_States(int16_t *contactorStates);
```

Comment:
Be consistent with the naming. So I would call this BSP_Contactors_Get() since it's standard to have setter and getter functions.

Reply:
Also, as the comment below this says, I would include the enum as a parameter here also and make the function actually return the state of that contactor instead of getting both into a buffer array.

Reply:
Changed to BSP_Contactors_Get()